.github/workflows/release.yml

```diff
@@ -0,0 +1,20 @@
+name: Release
+on:
+  push:
+    branches: [master]
+    tags: ["*"]
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
```

Comment:
don't know if it's important; `ci.yml` uses:
* `actions/checkout@v2`, not `v1`
* `olafurpg/setup-scala@v7`, not `v2`